components/advanced_text_editor/formatting_bar/formatting_icon.tsx

```diff
@@ -39,7 +39,12 @@ export const IconContainer = styled.button`
     padding: 0;
     border-radius: 4px;
     color: rgba(var(--center-channel-color-rgb), 0.56);
-    fill: currentColor;
+
+    > svg {
+        width: 18px;
+        height: 18px;
+        fill: currentColor;
+    }
```

Comment:
We can just remove that since the compass-icon components have these as props: `size` and `color`

```suggestion
```